### DISCUSSION TITLE
How do you make the PC as host for Ooba, then connect to Mobile with Sillytavern? (been trying for a week now to no avail)

### DISCUSSION THREAD

**Discussion by kuryuzzaky** [oobabooga/text-generation-webui#2823] [Q&A] [answered]
you guys make it sound so easyy to do T⌓T  (1st time ooba user here)

![Screenshot 2023-06-23 015546](https://github.com/oobabooga/text-generation-webui/assets/113837421/3aaafe1f-ef93-42f2-acab-337fa3a1a597)

turned off firewall on pc, antivirus as well. Mobile and pc connected to the same network, can Ooba make cloudflare link like koboldai? does IP settings which can be DHCP/Manual affect this? 

in my mobile, blocking api url is set to: http://127.0.01:5000/api
and for streaming api url:http://pc's_ivp4:port 

"Sillytavern is listening on all networks"

with the ooba subreddit still on private, cant access similar issues like this to solve it

what am I doing wrong? Thank you in advance

**Comment by syaoran68** [comment]
you'll need to broadcast on 0.0.0.0 for easiest compatibility.. i've added the '--listen --listen-host 0.0.0.0' and that should allow you to hit it. I would check first with another machine on the same network (wifi) if you can hit it then it might be a firewall / port forwarding issue.

**Reply by kuryuzzaky** [reply]
![IMG_20230623_123111](https://github.com/oobabooga/text-generation-webui/assets/113837421/2d91e394-dd11-4298-bc8a-bba86156ebb9)

Like this?

**Reply by kuryuzzaky** [reply]
Tbh, at this point, could really use a video tutorial

Tried experimenting with other mobile devices copying the sillytavern url 127.0.0.1:8000 (sillytavern is hosted by pc) which my mobile browser said "this site can't be reached, refused to connect"

**Reply by missionfloyd** [reply]
0.0.0.0 isn't an actual IP address. Use your PC's IP (whatever you crossed out above, not 127.0.0.1 or 0.0.0.0).

**Reply by kuryuzzaky** [reply, marked as answer]
Managed to fix it, not an ooba or sillytavern problem, its my old router causing port issues. Had to reset router and do some tinkering before i managed to get it to work. At first i used my actual ip and the port given by ooba but yeah, didn't worked cause of router porting issue. Thanks for replying nonetheless!
